analyzers/tests/SonarAnalyzer.UnitTest/Common/Paths.cs

```diff
@@ -22,18 +22,20 @@
 
 namespace SonarAnalyzer.UnitTest.Common
 {
-    public static class PathResolver
+    public static class Paths
     {
-        public static readonly string RelativePathToTestProjectRoot = @"..\..\..\";
+        public static string TestProjectRoot { get; }
 
-        static PathResolver()
+        static Paths()
         {
             // The AltCover tool has a limitation. It has to be invoked without a parameter for the project/solution path.
             // Due to this we have to call it from the analyzers folder and the working directory is different when running in CI context.
-            if (!File.Exists(Path.Combine(RelativePathToTestProjectRoot, "nuget.config")))
-            {
-                RelativePathToTestProjectRoot = @"..\..\..\..\";
-            }
+            var testProjectRoot = File.Exists(Path.Combine(@"..\..\..\", "nuget.config"))
+                                      ? @"..\..\..\"
+                                      : @"..\..\..\..\";
+
+            TestProjectRoot = Path.GetFullPath(testProjectRoot);
+            Console.WriteLine(@"Test project root: " + TestProjectRoot);
```

Comment:
I have mixed feelings about this `Console.WriteLine` 😅 But I guess it's fine here